pkg/provisioner/crdprovisioner.go

```diff
@@ -1228,3 +1253,24 @@ func startWatchingKube(ctx context.Context, kubeInformerFactory informers.Shared
 		os.Exit(1)
 	}
 }
+
+func startWatchingCrd(ctx context.Context, crdInformerFactory crdInformers.SharedInformerFactory, driverUninstall *uint32) {
+	crdInformer := crdInformerFactory.Apiextensions().V1().CustomResourceDefinitions().Informer()
+	crdInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: nil,
+		UpdateFunc: func(_, new interface{}) {
```

Comment:
Shouldn't we do this on delete rather than update? I don't believe we'd be guaranteed to see the update for the CRD deletion. There do not appear to be any finalizers, so deletion should be near instantaneous.